### PR TITLE
Option for not closing article after pressing outside article area

### PR DESCRIPTION
**Pull request by psukys**
Added additional configuration setting for #1530 . This includes default settings and translation entry for English.

### CODE REVIEW

**Comment by Alkarex** [comment]
Let's try to make some more translations (I will do a few)

File: app/i18n/en/conf.php [hunk 1]
```diff
@@ -93,6 +93,7 @@
 		'display_categories_unfolded' => 'Show categories folded by default',
 		'hide_read_feeds' => 'Hide categories & feeds with no unread articles (does not work with “Show all articles” configuration)',
 		'img_with_lazyload' => 'Use "lazy load" mode to load pictures',
+		'sides_close_article' => 'Clicking outside of article text area closes the article',
```
Comment (Alkarex) [inline app/i18n/en/conf.php:96]: Could you please add this line to all other languages, suffixed with a TODO?

```php
'sides_close_article' => 'Clicking outside of article text area closes the article', //TODO
```

**Comment by Alkarex** [comment]
Please add yourself to https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md

**Comment by psukys** [comment]
@Alkarex added the line to other languages with TODO comment. Also added myself to credits

**Comment by Alkarex** [comment]
Thanks @psukys !
Merged with a few changes. Please test /dev version if you can.

**Comment by Frenzie** [comment]
Awesome! I never liked that but my annoyance with it was extremely minor. :-)

**Pull request merged**